ambry-account/src/main/java/com/github/ambry/account/mysql/ContainerDao.java

```diff
@@ -53,6 +53,8 @@ public ContainerDao(MySqlDataAccessor dataAccessor) {
     getByAccountSql =
         String.format("select %s, %s, %s from %s where %s = ?", ACCOUNT_ID, CONTAINER_INFO, LAST_MODIFIED_TIME,
             CONTAINER_TABLE, ACCOUNT_ID);
+    updateSql = String.format("update %s set %s = ?, %s = 1, %s = now() where %s = ? AND %s = ? ", CONTAINER_TABLE,
```

Comment:
You don't want to set version = 1 on update.

Reply:
Sure, true. Added a todo to change it after adding version field in Container class. I think that we might need to add version field on both ambry and nuage-ambry at same time to avoid serialization/deserialization issues.